pkg/operation/secretutils/secretutils.go

```diff
@@ -0,0 +1,158 @@
+// Copyright (c) 2018 SAP SE or an SAP affiliate company. All rights reserved. This file is licensed under the Apache Software License, v. 2 except as noted otherwise in the LICENSE file
```

Comment:
Instead of creating a new package, can you move the contents of the `secretutils` package to `pkg/utils/secrets.go`?